pkg/dash/dash.go

```diff
@@ -544,8 +545,19 @@ func enableOpenCensus() error {
 
 // findKubeConfig looks for kube config from .kube or provided by user
 func findKubeConfig(logger log.Logger, kubeConfig string) error {
-	if _, err := os.Stat(kubeConfig); err == nil {
-		logger.Infof("using kube config: %v", kubeConfig)
+	found := false
+	paths := filepath.SplitList(kubeConfig)
+
+	for _, path := range paths {
```

Comment:
I would prefer to see this loop running and capturing the error for each kubeconfig passed in.
We can then exclude any configs we are unable to stat from the Watch and log the paths we failed to stat as errors.

Also, the comment in the PR reads that we will fail if any of the kubeconfig passed in are inaccessible. I don't think that is the behavior this code has, but also I think that is wrong behavior here, we can provide a good message around this. Consider something like:

```
Watching kubeconfig:
  - /path/to/config [SUCCESS]
  - /path/to/config [SUCCESS]
  - /path/to/config [FAILED]
```

Reply:
The failure happens later when creating a cluster client with the bad config path. If we want the behavior for octant to run if any of the provided paths are valid, `findKubeConfig` will have to return a filtered file list

Reply:
I don't think we should add the path when `os.Stat` fails.

Reply:
The watcher already shows what is being watched.
```
2020-07-29T09:50:50.811-0700    INFO    dash/watcher.go:116     watching config file    {"component": "config-watcher", "config": "/home/sfoo/.kube/config"}
2020-07-29T09:50:50.811-0700    INFO    dash/watcher.go:116     watching config file    {"component": "config-watcher", "config": "/home/sfoo/.kube/octant-dev-cluster"}
```
Instead logging will only show what can't be found